src/main/java/controller/LocationController.java

```diff
@@ -0,0 +1,55 @@
+package controller;
+
+import db.DatabaseHandler;
+
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.sql.Timestamp;
+
+public class LocationController {
+
+    DatabaseHandler databaseHandler = new DatabaseHandler();
+
+    public void listAllLocations() {
+        Statement statement = databaseHandler.createStatement();
+
+        try {
+            ResultSet resultSetLocations = statement.executeQuery("SELECT * FROM locations");
+
+            while (resultSetLocations.next()) {
+                int locationID = resultSetLocations.getInt("id");
+                String stopName = resultSetLocations.getString("stopName");
+                Timestamp createdOn = resultSetLocations.getTimestamp("createdOn");
+                System.out.println(locationID + " - " + stopName + " - " + createdOn);
+            }
+            statement.close();
+            databaseHandler.closeConnection();
+        } catch (SQLException ex) {
+            ex.printStackTrace();
+        }
+    }
+
+    public void findByLocationID(int id) {
+        Statement statement = databaseHandler.createStatement();
+
```

Comment:
will be good to have same text explanation in the output for more clear readability
![2020_02_28_20_38_50_2020_02_18_Database_in_github_D_JAVA_SDA_2 _Home_task_2020_02_18_Database_in_g](https://user-images.githubusercontent.com/56398857/75577573-f31cb700-5a6a-11ea-8cf4-80ac08abe47e.png)